Python/pywarpx/callbacks.py

```diff
@@ -93,8 +93,11 @@ def __call__(self,*args,**kw):
 
     def clearlist(self):
         self.funcs = []
```

Comment:
Let's maybe add a doc string here while we touch it:
```suggestion
        """Unregister/clear out all registered C callbacks"""
        self.funcs = []
```
(Pls double check if that is the correct doc string :) )